### PR TITLE
BugFixes

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
- 1068720: Email validation is needed and tool is required for the investigation period
- 1066095: Added running playbook web socket to wait unit the config playbook completed
- 1063967: Validation added on Connector health check
- 1063468: Added NIST connector by default in connector config page
- 1062918: Include outbreak for the next days should be restricted to 365
- 1059363: Switching tab for invalid config in connector config page

### CODE REVIEW

File: widget/view.controller.js [hunk 1]
```diff
@@ -8,19 +8,21 @@
...
+    outbreakAlertConfiguration200Ctrl.$inject = ['$scope', '$http', 'WizardHandler', '$controller', '$state', 'connectorService', 'marketplaceService', 'CommonUtils', '$window', 'toaster', 'currentPermissionsService', '_', '$resource', 'API', 'ALL_RECORDS_SIZE', 'widgetBasePath', '$rootScope', 'websocketService', '$timeout', 'widgetUtilityService'];
 
-    function outbreakAlertConfiguration200Ctrl($scope, $http, WizardHandler, $controller, $state, connectorService, marketplaceService, CommonUtils, $window, toaster, currentPermissionsService, _, $resource, API, ALL_RECORDS_SIZE, widgetBasePath, $rootScope, SchedulesService, $timeout, widgetUtilityService) {
+    function outbreakAlertConfiguration200Ctrl($scope, $http, WizardHandler, $controller, $state, connectorService, marketplaceService, CommonUtils, $window, toaster, currentPermissionsService, _, $resource, API, ALL_RECORDS_SIZE, widgetBasePath, $rootScope, websocketService, $timeout, widgetUtilityService) {
         $controller('BaseConnectorCtrl', {
             $scope: $scope
         });
         $scope.processingPicklist = false;
+        $scope.hutparams = {};
+        $scope.isConnectorsHealthy = false;
```
Comment (cs-pranali-chavan) [inline widget/view.controller.js]: isConnectorsHealthy change to isConnectorHealthy

File: widget/view.controller.js [hunk 2]
```diff
@@ -33,7 +35,9 @@
         $scope.toggle = [];
         $scope.toggleRemediation = true;
         $scope.toggleConnectorConfig = [];
```
Comment (cs-pranali-chavan) [inline widget/view.controller.js:37]: toggleConnectorConfig toggle keyword is for true/false 
please update the name

File: widget/view.controller.js [hunk 3]
```diff
@@ -68,19 +77,55 @@
...
+        var subscription;
+        initWebsocket();
+
+        $scope.$on('websocket:reconnect', function () {
+            initWebsocket();
+        });
+
+        function initWebsocket() {
+            websocketService.subscribe('runningworkflow', function (data) {
+                if (data.parent_wf === 'null' && $scope.isPlaybookExecuted === false) {
```
Comment (cs-pranali-chavan) [inline widget/view.controller.js:91]: null value should not be as string. Please check once

File: widget/view.controller.js [hunk 4]
```diff
@@ -349,47 +409,127 @@
...
             if (installationForm.notificationForm.$invalid) {
-                installationForm.notificationForm.fromEmailAddressa.$touched = true;
-                installationForm.notificationForm.fromEmailAddressa.$untouched = false;
-                installationForm.notificationForm.fromEmailAddressa.$dirty = true;
+                installationForm.notificationForm.fromEmailAddress.$touched = true;
+                installationForm.notificationForm.fromEmailAddress.$untouched = false;
+                installationForm.notificationForm.fromEmailAddress.$dirty = true;
                 return;
             }
-            WizardHandler.wizard('OutbreaksolutionpackWizard').next();
```
Comment (cs-pranali-chavan) [inline widget/view.controller.js:392]: Is this statement not required now?

File: widget/widgetAssets/locales/en.json [hunk 5]
```diff
@@ -38,7 +38,7 @@
         "FOURTH_PAGE_SECTION_1_DISCRIPTION": "Specify the number of days within which outbreak should be considered for investigation.",
         "FOURTH_PAGE_SECTION_1_PARAM_1": "Include outbreak for the next ",
         "FOURTH_PAGE_SECTION_1_PARAM_2": "Days",
-        "FOURTH_PAGE_SECTION_1_TOOLTIP_1": "Number should be greater than 0",
+        "FOURTH_PAGE_SECTION_1_TOOLTIP_1": "Only positive integers are allowed",
```
Comment (cs-pranali-chavan) [inline widget/widgetAssets/locales/en.json:41]: Description typo errror

**Review by cs-pranali-chavan — CHANGES_REQUESTED** [review]

File: widget/view.controller.js (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply widget/view.controller.js:37]: its save true/false in list format

File: widget/view.controller.js (hunk 4 above)
Comment (cs-sujeet-ahirrao) [reply widget/view.controller.js:392]: yes not required

File: widget/view.controller.js (hunk 3 above)
Comment (cs-sujeet-ahirrao) [reply widget/view.controller.js:91]: verified in UI code

**Review by cs-pranali-chavan — APPROVED** [review]
Looks good

**Pull request closed without merging**